### PR TITLE
fix: move `conf/cert` to `t/certs` and disable ssl by default

### LINKED ISSUE #2110: request help: move `conf/cert` to `t/cert`
all of these files are ONLY for test cases.

### PR DESCRIPTION
**Pull request by Yiyiyimu**
### What this PR does / why we need it:
fix #2110 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by membphis** [comment]
@Yiyiyimu please fix the issue of CI

**Comment by Yiyiyimu** [comment]
Hi @membphis the CI error is due to there're some problems unsolved. Discussion is on the original issue #2110.

**Comment by Yiyiyimu** [comment]
Kind of stuck hereand need some help. Thx~

File: conf/config-default.yaml [hunk 1]
```diff
@@ -61,7 +61,7 @@ apisix:
     - 127.0.0.0/24              # If we don't set any IP list, then any IP access is allowed by default.
   #   - "::/64"
   # port_admin: 9180              # use a separate port
-  https_admin: true             # enable HTTPS when use a separate port for Admin API.
+  # https_admin: true             # enable HTTPS when use a separate port for Admin API.
```
Comment (juzhiyuan) [inline conf/config-default.yaml:64]: What's the default value?

**Comment by nic-chen** [comment]
@Yiyiyimu 

It is another issue now. 

I think it is caused by ssl not being enabled by default:
https://github.com/apache/apisix/pull/2112/files#diff-8d872babc717e9d733641b56bfc530ef98751fbe4e68f08d79b2b83109c22fffR105

You can modify the test case accordingly and move on.

File: conf/config-default.yaml (hunk 1 above)
Comment (Yiyiyimu) [reply conf/config-default.yaml:64]: nil?

**Comment by Yiyiyimu** [comment]
Thank you for the fix and problem pointing out @nic-chen! All test passed and ready to be reviewed~

File: conf/config-default.yaml [hunk 2]
```diff
@@ -101,12 +101,15 @@ apisix:
...
   ssl:
-    enable: true
+    enable: false                 # ssl is disabled by default
+                                  # enable it to use your own cert and key
     enable_http2: true
     listen_port: 9443
     # ssl_trusted_certificate: /path/to/ca-cert # Specifies a file path with trusted CA certificates in the PEM format
                                                 # used to verify the certificate when APISIX needs to do SSL/TLS handshaking
                                                 # with external services (e.g. etcd)
+    ssl_cert: ""
```
Comment (moonming) [inline conf/config-default.yaml:111]: Here is the path or content of the certificate? we need to add comments.

**Review by moonming — APPROVED** [review]

**Comment by moonming** [comment]
@nic-chen please take a look

File: conf/config-default.yaml (hunk 2 above)
Comment (Yiyiyimu) [reply conf/config-default.yaml]: Thank you! fixed

**Review by nic-chen — APPROVED** [review]

**Pull request merged**